src/bms/parse/prompt.rs

```diff
@@ -4,12 +4,9 @@
 
 use std::path::Path;
 
-use crate::{bms::Decimal, lex::command::ObjId};
+use crate::bms::{Decimal, command::*};
 
-use super::{
-    ParseWarning, Result,
-    header::{AtBgaDef, BgaDef, Bmp, ExRankDef, ExWavDef},
-};
+use super::{ParseWarning, Result, model::def::*};
```

Comment:
Please don't use wildcard import in the file using cfg feature attributes. It may be introduced namespace collision, especially switched feature flags.